orc8r/cloud/go/tools/accessc/handlers/add_admin_token.go

```diff
@@ -0,0 +1,78 @@
+package handlers
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"magma/orc8r/cloud/go/services/certifier"
+	certprotos "magma/orc8r/cloud/go/services/certifier/protos"
+	"magma/orc8r/cloud/go/tools/commands"
+)
+
+func init() {
+	cmd := CommandRegistry.Add(
+		"add-admin-token",
+		"Add a new admin which has all permissions to read/write all entities and create a new admin token",
+		addAdminToken)
+
+	f := cmd.Flags()
+	f.Usage = func() {
+		fmt.Fprintf(os.Stderr, // std Usage() & PrintDefaults() use Stderr
+			"\tUsage: %s %s [OPTIONS] <Admin Username> <Admin Password>\n", os.Args[0], cmd.Name())
```

Comment:
This is non-standard line-breaking. It's (almost) always:

- Open and close paren each on their own line, at same level of indentation
- Each arg on their own line